docker/developers/docker-compose.dashboard.yml

```diff
@@ -0,0 +1,32 @@
+include:
+  - path: ./docker-compose.mysql.yml
+  - path: ./docker-compose.redis.yml
+  - path: ./docker-compose.networks.yml
+
+services:
+  # Runs all the dependent services when the dashboard server is running natively
+  dashboard-services:
+    image: alpine
+    depends_on:
+      mysql:
+        condition: service_healthy
+      redis:
+        condition: service_started
+    command: >
+      /bin/sh -c "
+        echo 'Dashboard services running.' &&
+        echo &&
+        echo 'To shut down all containers:' &&
+        echo 'docker compose stop' &&
+        echo &&
+        echo 'To run the native dashboard server:' &&
+        echo './bin/dashboard-server' &&
+        echo &&
+        echo 'Be sure to have the following in your locals.yml to link to the contained services:' &&
+        echo &&
+        echo 'db_writer: mysql://root:password@127.0.0.1/' &&
+        echo 'db_endpoint_writer: 127.0.0.1' &&
+        echo 'db_credential_admin:' &&
+        echo '  username: user' &&
```

Comment:
Is this intentionally not `root`? Introduced a bit of confusion as our `mysql-client-admin` script uses these credentials and doesn't end up with full visibility into schema/tables created with the `db_writer` credentials.

Reply:
That seems true that it could be `root` here.